torch/csrc/jit/runtime/symbolic_script.cpp

```diff
@@ -342,7 +378,8 @@ const std::vector<std::string> functions = {
                 grad_mat2 = AD_bmm_backward_mat2(grad_output, self)
                 return grad_self, grad_mat2
             return torch.bmm(self, mat2), backward
-
+    )",
+    R"(
```

Comment:
What are these two lines for? I'm confused.

Reply:
There's a max string literal size on MSVC. So, I'm splitting the string to get below the size limit.